### ISSUE TITLE
talk-Writing Dev-Friendly tests 

### ISSUE THREAD

**Issue by ameerthehacker** [jslovers/Welcome-to-JSLovers#8]
---
 ## Writing Dev-Friendly tests

Make tests your Allies to fight the bug war before it hits the production

 ## Description

The first thing that comes to our mind when we think about testing our JavaScript is that it is painful and time-consuming. In my talk, I would like to share what we can do to make it more developer-friendly, effective and yes enjoyable

 Things covered in the talk

* The golden rules
* Is the test pyramid still valid?
* Measuring the effectiveness of the test, get the answer for the question (Did I waste my entire time writing this test?)
* Immediate feedback to the developer by not slowing him down

 ## Prerequisite
* [ ] Basic knowledge on automated tests
* [ ] Basic idea on tools like jest

 ⌛️ Talk Duration - **45 mins**
👤 Name - Ameer Jhan
📍 Location - Bengaluru

**Comment by reeversedev** [comment]
Can you try and give a basic introduction of testing to people? I think that will make Audience more engaged in your talk.

**Comment by ameerthehacker** [comment]
Yes definitely I will be doing that in the best way possible to show them how enjoyable and easy is automated testing

**Comment by Neha** [comment]
@ameerthehacker  Could you please add the title and share the slides. We can look into October for the Meetup. Please let me know your availability.

**Comment by ameerthehacker** [comment]
Sure @Neha october totally works for me. I will send over the presentation, which email should I send it to?

**Comment by Neha** [comment]
@ameerthehacker  hello@jslovers.com

**Comment by ameerthehacker** [comment]
Hi @Neha I have shared the ppt as editable to the hello@jslovers.com

**Comment by ameerthehacker** [comment]
Hi @Neha anything else needed from my side?

**Comment by reeversedev** [comment]
@ameerthehacker No. I think this information is sufficient enough. Will add your twitter and linkedin in meetup description in a while.

**Comment by ameerthehacker** [comment]
cool thanks @reeversedev
